### ISSUE TITLE
Rescaling the images

### ISSUE THREAD

**Issue by fifi1ous** [graphdeco-inria/gaussian-splatting#738]
It's not an issue but it is more a question about some parts of the process.
One of the initial steps is to rescale the image into 1.6k x 1.6k, I assume that this step is because of the afterwards rasterizer process when they split the screen into 16x16 pixel tiles. I wonder why they chose the 16x16 pixel tiles? 
If someone chooses not to resize the images into different sizes than 1.6k x 1.6k, how does it affect the use of 16x16 pixel tiles? Also, what happens if the image dimensions aren't divisible by 16?
Lastly, could you tell me what kind of resizing algorithm you use?

**Comment by jaco001** [comment]
For training  GS uses pic at 1600 x yyyy (only  one size is 1600,  yyyy  is ratio dependent.)  by default. You can choose any image size with -r parameter if you have enough VRAM.

**Comment by fifi1ous** [comment]
And is there any way to find what rescaling algorithm GS uses? And what if the final yyyy isn't divisible by 16?

**Comment by jaco001** [comment]
![3910](https://github.com/graphdeco-inria/gaussian-splatting/assets/67650086/50298188-c9d9-4bb4-ad75-69c487e77ef5)
![3910](https://github.com/graphdeco-inria/gaussian-splatting/assets/67650086/6655bc19-3574-4758-a213-2c44d8a97a2c)

Some hint there ;) Do math
from 1920x1020 to 1600x850 ratio is (1,882352941176471)

**Comment by fifi1ous** [comment]
Yeah, I got this, but with the divisions I am talking about the rasterizer, which splits the screen into 16x16 tiles. Or am I getting it wrong and they dont split the picture screen, but the model on the screen.
